src/systems/brush.js

```diff
@@ -229,6 +245,19 @@ AFRAME.registerSystem('brush', {
 
     return stroke;
   },
+  createStrokeId: function () {
+    return Math.random().toString(36).substring(2, 9);
+  },
+  addPointToStroke: function (strokeId, data) {
+    var stroke;
+    for (var i = 0; i < this.strokes.length; i++) {
```

Comment:
@fernandojsg I removed the need for a strokeId and looping through each tick. Now the sync.js system keeps track of the last stroke for each user: https://github.com/aframevr/a-painter/pull/207/commits/b9bdb9e9d9352528e1576e73683d9cc877e9474b